imap_processing/cdf/cdf_attribute_manager.py

```diff
@@ -148,7 +154,7 @@ def add_global_attribute(self, attribute_name: str, attribute_value: str) -> Non
         self.global_attributes[attribute_name] = attribute_value
 
     @staticmethod
-    def _load_yaml_data(file_path: str | Path) -> dict:
+    def _load_yaml_data(file_path: str | Path) -> yaml:
```

Comment:
It does return a dict that is structured like yaml. (`dict` where the keys are strings and the values are dicts)